tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
```

Comment:
I'd like to see `strict: true` here here....It's best to start with `strict` so you don't have to go back and fix issues when you want to enable it later.

You can see the config we use for the [Juju Dashboard here](https://github.com/canonical-web-and-design/jaas-dashboard/blob/main/tsconfig.json)